specs/proxi-specifications.adoc

```diff
@@ -186,33 +172,31 @@ http://www.ebi.ac.uk/pride/archive/proxi/v1/datasets                         //G
 http://www.ebi.ac.uk/pride/archive/proxi/v1/datasets/{accession}             //Get datasets for given Accession
 ```
 
-[[filtering-collections]]
-=== Filtering collections
+[[compact-fullobejcts]]
+=== Compact vs Full Objects
 
-7. We RECOMMENDED to use query component to filter URI collection
+Every API collection point SHOULD provide a parameter called `resultType`. This parameter will define the type of the object the API call will retrieve [Full or Compact].
 
-Many times, you will come across requirements where you will need a collection of resources sorted,
-filtered or limited based on some certain resource attribute.
+- A Compact object is a version of the object that only contains the required fields of the Object type. For example, for a peptide spectrum match would be the `peptideSequence` or for an spectrum the `unified spectrum identifier`.
 
-For this, do not create new APIs – rather enable sorting, filtering and pagination capabilities in resource
-collection API and pass the input parameters as query parameters:
+- A Full object is a version of the object that contains all the attributes/fields of the Object type.
 
-```
-http://www.ebi.ac.uk/pride/archive/proxi/v1/datasets?species=Human
-```
+6. We RECOMMENDED the use of **Compact** version of an object as default behavior for collections end-points, for example:
 
-[[compact-full-objects]]
-==== Compact vs Full Object versions
+[[filtering-collections]]
+=== Filtering collections
 
-8. We RECOMMENDED the use of **Compact** version of an object as default behavior for collections end-points.
+7. We RECOMMENDED to use query component to filter URI collection
 
-Each object in the API MUST provide two flavours (Compact or Full). The **Compact** version of an object will be used when retrieve collections of the Object, for example:
+Many times, you will come across requirements where you will need a collection of resources `filtered` or limited based on some certain resource attribute. For this, do not create new APIs – rather enable filtering in resource collection API and pass the input parameters as query parameters:
 
 ```
-http://www.ebi.ac.uk/pride/archive/proxi/v1/datasets
+http://www.ebi.ac.uk/pride/archive/proxi/v1/datasets?species=Human
 ```
 
-The Compact version will contains all the mandatory attributes for an Object. When designing each Data Object, mandatory fields should be the Minimum Information requeried to identified the object.
+The filtering will be defined as the <propertyName>=<propertyValue> where the propertyName is the name of the attribute that would be use for filtering and the propertyValue is the possible value the user would like to filter the collection.
```

Comment:
1. Remove "the" before "&lt;propertyName&gt;=&lt;propertyValue&gt;", since it's not a specific noun requiring distinction via the use of a definite article.
2.  Change "use" to "used" since this is a passive infinitive requiring the past participle of the verb.
3. Remove "possible". There is no possibility here; the collection will be filtered by this value.
4.  The end of this sentence is grammatically ambiguous. Clarify with something like "the value **by which** the user would like to filter" or  "the value the user would like to filter the collection **by**". The latter is technically a split infinitive but it sounds less awkward conversationally, whereas the former is more grammatically correct but sounds a bit formal.